onnxruntime/python/tools/transformers/convert_beam_search.py

```diff
@@ -128,6 +128,17 @@ def parse_arguments(argv=None):
                                    default=1,
                                    help='Positive. >1 to penalize and <1 to encorage.')
 
+    beam_search_group.add_argument('--vocab_size',
```

Comment:
Do not need vocab_size input. It can be deduced from GPT-2 model.

Reply:
vocab size is used while creating masks. Model vocab size can vary based on model. For instance, deep suggest is trained with 50297 whereas GPT2 is 50257.

Reply:
It is possible to get vocab_size from shape of logits output of the onnx model.